src/operator/subgraph/dnnl/dnnl_remove_casts_property.h

```diff
@@ -137,6 +137,15 @@ class SgDNNLRemoveCastsProperty : public SubgraphProperty {
     auto selector = std::make_shared<SgDNNLRemoveCastsSelector>();
     return selector;
   }
+
+  void ConnectSubgraphOutputs(const nnvm::ObjectPtr subgraph_node,
+                              std::vector<nnvm::NodeEntry*>* output_entries) const override {
+    // Connect all extern output entries to output[0]
+    for (size_t i = 0; i < output_entries->size(); ++i) {
+      auto entry_ptr = output_entries->at(i);
```

Comment:
seems like entry_ptr->index is always 0 as cast has only one output - can we simplify this? or comment above should be changed